pallets/runtime/common/src/runtime.rs

```diff
@@ -602,12 +602,29 @@ macro_rules! runtime_apis {
             Runtime,
             AllPalletsWithSystem,
             (
-              pallet_preimage::migration::v1::Migration<Runtime>,
-              pallet_scheduler::migration::v3::MigrateToV4<Runtime>,
```

Comment:
Note: these are a one-time use migration.  Testnet and Mainnet have already run these, so we don't need them anymore.